charts/plugin-br-pix-switch/templates/ingress-providers.yaml

```
@@ -0,0 +1,63 @@
+{{- if .Values.providersIngress.enabled }}
+{{- $ctx := . }}
+{{- $cfg := .Values.providersIngress }}
+{{- $fullName := include "plugin-br-pix-switch.componentFullname" (dict "context" $ctx "component" "providers") }}
+{{- $renderedRoutes := list }}
+{{- range $route := $cfg.routes }}
+  {{- $componentValues := index $ctx.Values $route.component }}
+  {{- if and $componentValues $componentValues.enabled }}
+    {{- $renderedRoutes = append $renderedRoutes $route }}
+  {{- end }}
+{{- end }}
+{{- if $renderedRoutes }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)